kinto/core/storage/redis.py

```diff
@@ -342,11 +356,14 @@ def delete_all(self, collection_id, parent_id, filters=None,
                    modified_field=DEFAULT_MODIFIED_FIELD,
                    deleted_field=DEFAULT_DELETED_FIELD,
                    auth=None):
+
         records, count = self.get_all(collection_id, parent_id,
                                       filters=filters,
                                       id_field=id_field,
                                       modified_field=modified_field,
                                       deleted_field=deleted_field)
+        # XXX: Here we need to have the correct coolection_id and
+        # parent_id for the record id
```

Comment:
Yeah, I am currently blocked there, I have two solutions, either I am finding a way to give collection_id and parent_id to delete, either I am rewritting ``delete_all``

Would you like to chat a little bit about it with me?